src/site/markdown/error_handling.md

```diff
@@ -0,0 +1,4 @@
+# Error Handling
```

Comment:
What is the point of this file?  Is it linked from anywhere?  It also links to itself which I don't understand.